config/src/config/storage_config.rs

```diff
@@ -19,8 +19,9 @@ pub struct StorageConfig {
 impl Default for StorageConfig {
     fn default() -> StorageConfig {
         StorageConfig {
-            address: "localhost".to_string(),
-            port: 6184,
+            address: "127.0.0.1:6184"
+                .parse()
+                .expect("Unable to parse socket address"),
```

Comment:
I think in some of these cases and unwrap is good enough.

Reply:
Done, thanks!